packages/fxa-settings/src/components/Settings/SubRow/en.ftl

```diff
@@ -12,4 +13,22 @@ tfa-row-backup-codes-get-new-cta = Get new codes
 # Button to add backup authentication codes when none are configured
 tfa-row-backup-codes-add-cta = Add
 # 'This' refers to 'backup authentication codes', used as a recovery method for two-step authentication
-tfa-row-backup-codes-description = This is the safest recovery method if you canʼt access your mobile device or authenticator app.
+tfa-row-backup-codes-description-2 = This is the safest recovery method if you canʼt use your mobile device or authenticator app.
+
+# Backup recovery phone is a recovery method for two-step authentication
+# A recovery code can be sent to the user's phone
+tfa-row-backup-phone-title = Backup recovery phone
+# Shown with an alert icon to indicate that no backup recovery phone is configured
+tfa-row-backup-phone-not-available = No recovery phone number available
+# button to change the configured backup recovery phone
+tfa-row-backup-phone-change-cta = Change
+# button to add/configure a backup recovery phone
+tfa-row-backup-phone-add-cta = Add
+# Button to remove a backup recovery phone from the user's account
+tfa-row-backup-phone-delete-button = Remove
+# Shown in tooltip on delete button or delete icon
+tfa-row-backup-phone-delete-title = Remove backup recovery phone
+tfa-row-backup-phone-delete-restriction = If you want to remove your backup recovery phone, add backup authentication codes or disable two-step authentication first to avoid getting locked out of your account.
+# "this" refers to backup recovery phone
+tfa-row-backup-phone-description = This is the easier recovery method if you canʼt use your authenticator app.
+tfa-row-backup-phone-sim-swap-risk-link = Learn about SIM swap risk
```

Comment:
There's a chance localizers might not understand `SIM swap`. Any chance we could include the link/explanation in a comment?

Reply:
We don't have the article yet, but I can add a short explanation in comment.